DirectEmployers/MyJobs
# [PD 2829] date display current range

Task was to display the current date range in the calendar header

To Test:

Pull down the repository. You can see the current date range selected in the top Calendar section. The date range is specific per each tab so if you switch it and then switch between tabs, the current date range selected for each tab will show in the top of the calendar.
